locales/ara.json

```diff
@@ -64,12 +66,12 @@
     "ok": "تمام",
     "errors": {
       "savingFailed": {
-        "title": "فشل حفظ {{vcLabelSingular}}",
```

Comment:
Are we not replacing the {{placeholder}} with the {{native_translations}} ? For Arabic alone, the order itself is changed. any specific reason for arabic ?